### DISCUSSION TITLE
如何设置参考文献最后的列表之中，中文在前，英文在后？

### DISCUSSION THREAD

**Discussion by xiaoqiangch** [nju-lug/NJUThesis#226] [General]
如何设置参考文献最后的列表之中，中文在前，英文在后？

我试着修改def文件，但是被告知sorting=cenyt是无效的

<img width="472" alt="image" src="https://github.com/nju-lug/NJUThesis/assets/16594186/9913130b-bd63-4b7b-aaca-8df8f0622c6a">

**Comment by hushidong** [comment]
默认就是中文在前的吧，不用加sorting选项的，加选项gblanorder=chineseahead试试
